src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { getMovie, expeсtedMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard/MovieCard';
+
+interface FindMovieProps {
+  handleAddMovie: (movie: Movie) => void,
+}
+
+export const FindMovie: React.FC<FindMovieProps> = ({ handleAddMovie }) => {
+  const [search, setSearch] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [hasError, setHasError] = useState(false);
+
+  const handleSearch = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setSearch(value);
+    setHasError(false);
+  };
+
+  const handleFindMovie = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsLoading(true);
+    getMovie(search)
+      .then(result => {
+        if ((result as MovieData).imdbID) {
+          setMovie(expeсtedMovie(result as MovieData));
+        } else {
```

Comment:
let's just set in api that this function will return `MovieData` instead of those assertions